pkg/ccl/backupccl/file_sst_sink.go

```diff
@@ -80,6 +80,10 @@ type fileSSTSink struct {
 	}
 }
 
+// logicalBackupFileSize is the maximum logical size of a backup file that can
+// be extended.
+const logicalBackupFileSize = 64 << 20 // 64MB
```

Comment:
@dt given the interaction with midRow logic, do you still prefer hardcoding this?

Reply:
nit: "fileSize" is pretty confusing here, since we have "files" that have a "size" limit and it isn't this.

How about `fileSpanByteLimit` since what we're talking about is limiting spans in those files?

Reply:
Also personal preference, but I avoid the `// 64MB` comments --  a) they're not the norm in the repo, b) they can end up wrong and misleading if the const is updated without updating the comment. For the three easy to remember ones (10/20/30 = kb/mb/gb) I think we usually don't include them.

Reply:
Yeah, not here (and not for another couple weeks until we're sure we're done with backports), but we should just rename manifest's File to "ExportedSpan", and "Spans" to "CoveredSpans"